phys/module_physics_init.F

```diff
@@ -49,6 +49,7 @@ SUBROUTINE phy_init ( id, config_flags, DT, restart, zfull, zhalf,     &
                          re_cloud, re_ice, re_snow,              & ! G. Thompson
                          has_reqc, has_reqi, has_reqs,           & ! G. Thompson
 #if ( EM_CORE == 1 )
+                         PRES,RHOA,CCNTY,QDCN,QTCN,QCCN,QRCN,QNIN,& ! for ntu3m
```

Comment:
@tzuchin12 
Break this into two lines to allow the conventional spacing at the end of the line

Reply:
Modified as suggested.